### PR TITLE
METRON-600: Fix Metron Website

### PR DESCRIPTION
**Pull request by iraghumitra**
1.Updated the apache logo with a better resolution icon
2.Added License section to About page, the license section has a link to Apache License page
3.Added Mailing List Section to Community page. Discussion and Docs change mailing lists are added in this section
4.Added Downloads section in documentation. The link points to the latest release (The design is simlar to what Storm has)
5.Added icubator logo to top right of the header
6.Fixed the alignment issues in Firefox

### CODE REVIEW

**Comment by ottobackwards** [comment]
Tested with jekyll serve and safari.  Everything seems to work.  I am wondering about the redundancy between the community resources and getting involved links and pulling the mailing list out of those pages to the main page however.

Holding my +1 for comments.

File: site/documentation/index.md [hunk 1]
```diff
@@ -69,3 +70,29 @@ title: Apache Metron (Incubating) Documentation
...
+                </a>
+                 [ <a href="http://www.apache.org/dyn/closer.cgi/incubator/metron/0.3.0/apache-metron-0.3.0-incubating.tar.gz.asc" target="new">
+                    PGP
+                 </a>]
+                 [ <a href="http://www.apache.org/dyn/closer.cgi/incubator/metron/0.3.0/apache-metron-0.3.0-incubating.tar.gz.sha" target="new">
+                    SHA512
+                 </a>]
+                 [ <a href="http://www.apache.org/dyn/closer.cgi/incubator/metron/0.3.0/apache-metron-0.3.0-incubating.tar.gz.md5" target="new">
+                    MD5
+                  </a>]
```
Comment (mattf-horton) [inline site/documentation/index.md]: Sorry, but as stated in the Jira:
- All links to the downloadable distribution artifacts MUST NOT reference the main Apache dist web site (dist.apache.org). Instead, they should use the standard scripting mechanisms to distribute the load between the mirror sites (see http://www.apache.org/dyn/closer.cgi/ ).
- All links to checksums, detached signatures and public keys MUST reference the main Apache dist web site (dist.apache.org), and use https (SSL).

Therefore, this section should read:
```
 +                <a href="http://www.apache.org/dyn/closer.cgi/incubator/metron/0.3.0/apache-metron-0.3.0-incubating.tar.gz" target="new">
 +                    apache-metron-0.3.0-incubating.tar.gz
 +                </a>
 +                 [ <a href="https://dist.apache.org/repos/dist/release/incubator/metron/0.3.0/apache-metron-0.3.0-incubating.tar.gz.asc" target="new">
 +                    PGP
 +                 </a>]
 +                 [ <a href="https://dist.apache.org/repos/dist/release/incubator/metron/0.3.0/apache-metron-0.3.0-incubating.tar.gz.sha" target="new">
 +                    SHA512
 +                 </a>]
 +                 [ <a href="https://dist.apache.org/repos/dist/release/incubator/metron/0.3.0/apache-metron-0.3.0-incubating.tar.gz.md5" target="new">
 +                    MD5
 +                  </a>]
```
The first item (the tarball) is correct, the other three must use https and point at the dist.apache.org site.
Thanks for doing this.

**Comment by iraghumitra** [comment]
@mattf-horton My bad missed the point about checksums.

@ottobackwards The original idea was "Community Resources" would help people to follow metron activities and understanding metron in depth, "Get Involved" would help users to setup Metron and get it running. The current links might not be doing justice to this thought. 
I looked at storm, atlas and ranger websites none of them seem to be having the mailing list on the home page. The mailing list  is either on  "Getting Help" or on "Project Information" pages. We can move the mailing list to home page if we think it is useful there.

**Comment by james-sirota** [comment]
@mattf-horton does this adequately meet your concerns?

**Comment by mattf-horton** [comment]
Yes, +1.  I tested the links used, they work, and are in the right domains.

**Comment by cestella** [comment]
+1 by inspection

**Pull request closed without merging**